blog/authors.yml

```diff
@@ -1,23 +1,38 @@
-vimayya:
+mayya:
   name: Vinaya Mayya
   title: Senior Technical Program Manager - API
   url: https://www.ed-fi.org/success-stories/category/vinaya-mayya/
   image_url: https://www.ed-fi.org/assets/2019/11/Vinaya-Mayya-768x768.png
+  page: true
 
 hoekstra:
   name: Jason Hoekstra
   title: Senior Program Manager - Tools
   url: https://www.ed-fi.org/success-stories/category/jason-hoekstra/
   image_url: https://www.ed-fi.org/assets/2019/11/Jason-Hoekstra-2022-768x768.png
+  page: true
 
-sfuqua:
+fuqua:
   name: Stephen A. Fuqua
   title: Director, Software Engineering
   url: https://www.ed-fi.org/success-stories/category/stephen-fuqua/
   image_url: https://www.ed-fi.org/assets/2020/10/Stephen-Fuqua-768x768.png
+  page: true
+  socials:
+    linkedin: stephenfuqua
+    github: stephenfuqua
```

Comment:
@vimayya @jasonh-edfi @MrEdData please feel free to submit a pull request adding social links like this.